ios_device/cli/base.py

```diff
@@ -4,7 +4,7 @@
 from distutils.version import LooseVersion
 
 from ios_device.servers.DTXSever import DTXServerRPCRawObj, DTXEnum
-from ios_device.servers.Installation import InstallationProxyService
+from ios_device.servers.Installation import InstallationProxyServiceService
```

Comment:
Cannot find reference InstallationProxyServiceService

Reply:
Done, seems like I didn't check all the replaces